token-metadata/program/src/lib.rs

```diff
@@ -0,0 +1,12 @@
+//! A metadata program for the Solana blockchain.
+
+pub mod entrypoint;
+pub mod error;
+pub mod instruction;
+pub mod processor;
+pub mod state;
+pub mod utils;
```

Comment:
Does the `utils` module really need to be exported to downstream crates?
```suggestion
mod utils;
```